package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -274,46 +275,63 @@ def __init__(self, filename, convert_units=True, **kwargs):
         self.units['velocity'] = kwargs.pop('velocityunit',
                                  self.units['length']+'/'+self.units['time'])
 
-    def _write_atoms(self, atoms, data):
+    def _write_atoms(self, atoms, data, atom_style):
         self.f.write('\n')
-        self.f.write('Atoms\n')
+        self.f.write('Atoms  # ' + atom_style + '\n')
         self.f.write('\n')
 
         try:
             charges = atoms.charges
+            has_charges = True
         except (NoDataError, AttributeError):
             has_charges = False
-        else:
-            has_charges = True
 
         indices = atoms.indices + 1
         types = atoms.types.astype(np.int32)
 
-        moltags = data.get("molecule_tag", np.zeros(len(atoms), dtype=int))
+        if atom_style == 'full':
```

Comment:
Need to test for valid atom styles, I would add them as a static list object somewhere in this file.  e.g 

```python

_allowed_atom_styles = ["full, atomic, ..."]

if atom_style not in _allowed_atom_styles:
    raise ValueError("blah")
```